README.md

```diff
@@ -318,6 +318,8 @@ To use example app you need to first run below command
 cd example && npx react-native-asset
 ```
 
+> Note: Before doing npx react-native-asset, delete the file link-assets-manifest.json from the android and ios folders if it exists.
```

Comment:
@nilesh-simform Let's revise this note as follows
Note: If link-assets-manifest.json file already exists then make sure to delete that before running npx react-native-asset command